packages/sit-onyx/src/i18n/locales/en-US.json

```
@@ -37,5 +37,6 @@
     "goToHome": "Go to home page",
     "goBack": "Go back",
     "userMenuLabel": "User options"
-  }
+  },
+  "infoTooltip": "Info Tooltip"
```

```suggestion
  "infoTooltip": "Show info tooltip"
```

(maybe we should also rename the key to "showInfoTooltip")?